endpoints/patreon.js

```diff
@@ -0,0 +1,27 @@
+const { validatePatreonIdentity, logErrors } = require('../util');
```

Comment:
May be missing it, but I don't see that this still sends webhooks to discord anymore, doesn't use /utils/sendPatronWebhook.js

Reply:
Nevermind, it's in db stuff 👍